website/docs/install-deploy/deploying-distributed-cluster.md

```diff
@@ -70,58 +70,81 @@ cd fluss-$FLUSS_VERSION$/
 
 ### Configuring Fluss
 
-After having extracted the archived files, you need to configure Fluss for the cluster by editing `conf/server.yaml`
+After having extracted the archived files, you need to configure Fluss for the cluster.
+
+On **each node**, we will use the _default config file_ (`conf/server.yaml`) to configure the _TabletServer_.
+Additionally, we will use a separate _custom config file_ (`conf/coordinator-server.yaml`) to configure the _CoordinatorServer_ on **Node1**.
+The custom config file on **Node1** is necessary to avoid a port collision for the `bind.listeners` config option with between the CoordianatorServer and the TabletServer instance running on the same node.
```

Comment:
How about suggesting to use 4 nodes for the distributed deployment. This can make the deployment simple (each use its own `server.yaml`). Besides, this is also the recommended way for production purpose that the machine type of coordinator node can be different from tablet server node. Usually, tablet server should have a large local SSD disk or large EBS  (e..g, 4 vCPU 32 GiB 1TB), but coordinator doesn't need to have a local SSD disk, a small EBS is enough.

Reply:
Others looks good to me.